src/tests/rnp_tests.cpp

```diff
@@ -212,9 +212,10 @@ main(int argc, char *argv[])
       cmocka_unit_test(test_ffi_key_generate_dsa),
       cmocka_unit_test(test_ffi_key_generate_ecdsa),
       cmocka_unit_test(test_ffi_key_generate_eddsa),
-      cmocka_unit_test(test_ffi_key_generate_sm2),
+      cmocka_unit_test(test_ffi_key_generate_sm2),*/
       cmocka_unit_test(test_ffi_key_generate_ex),
-      cmocka_unit_test(test_ffi_add_userid),
+      cmocka_unit_test(test_ffi_key_generate_protection),
+      /*cmocka_unit_test(test_ffi_add_userid),
```

Comment:
Let's make sure we don't merge this with tests commented out.

Reply:
@dewyatt Wow, thanks, my fault. Commenting those out to make tests go faster, but this time missed to fix things back.

Reply:
@dewyatt Fixed commented out tests. Any other comments? Anyway waiting for CI.